src/Core/Contracts/JsonParser.ts

```diff
@@ -0,0 +1,4 @@
+export interface JsonParser {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  parse(text: string): any;
```

Comment:
Maybe `Record<string, unknown>` instead of `any`?

Reply:
My first attempt was `unknown`. But the native json.parse method returns the any type.
So this way it is more inline with what was already there.

Also, the api layer does not do any data structure validation.
So currently the `any` type does not complain when we actually want to return a domain type.
If we introduce `unknown` now, we would be typecasting anyway from unknown to an unvalidated assumed domain type.
I agree we should stay away from `any` as much as possible.
But I'd rather introduce a change like this explicitly with a ticket that has a goal of improving the type safety of the api layer.